<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InssuranceDialog</class>
 <widget class="QDialog" name="InssuranceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insurance</string>
  </property>
  <widget class="QComboBox" name="insuranceComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy Insurance</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ownedListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>119</y>
     <width>370</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
